static/js/compose.js

```diff
@@ -453,6 +460,7 @@ export function initialize() {
         const sub = stream_data.get_sub(stream_name);
         stream_settings_ui.sub_or_unsub(sub);
         $("#compose-send-status").hide();
+        finish({send_anyway: true});
```

Comment:
We likely want to change the label for this button if we're doing this change -- and of course let's keep this atomic change in a separate commit.

Reply:
DONE: I changed the button label to "Sending..." while the send-message is in progress, and then I restore the text when it succeeds or fails.

Reply:
Is there a UX change here that I should understand? I wasn't able to follow the comments above.